lua/telekasten.lua

```diff
@@ -448,21 +463,30 @@ local function create_note_from_template(
     end
 
     -- now write the output file, substituting vars line by line
-    local ofile = io.open(filepath, "a")
-    for _, line in pairs(lines) do
-        ofile:write(
-            templates.subst_templated_values(
-                line,
-                title,
-                calendar_info,
-                uuid,
-                M.Cfg.calendar_opts.calendar_monday
-            ) .. "\n"
-        )
-    end
+    local file_dir = filepath:match("(.*/)") or ""
+    check_dir_and_ask(file_dir, file_dir, function(dir_succeed)
+        if dir_succeed == false then
+            return
+        end
+
+        local ofile = io.open(filepath, "a")
+
+        for _, line in pairs(lines) do
+            ofile:write(
+                templates.subst_templated_values(
+                    line,
+                    title,
+                    calendar_info,
+                    uuid,
+                    M.Cfg.calendar_opts.calendar_monday
+                ) .. "\n"
+            )
+        end
```

Comment:
this is the bit that asks to create non-existent directories from the note title